src/olympia/devhub/templates/devhub/verify_email.html

```diff
@@ -20,21 +21,10 @@ <h1>{{ title }}</h1>
   Could not verify email address. The verification link has expired.
   {% endtrans %}
 {% elif state == "verification_pending" %}
-  {% trans %}
-  Working... Please be patient.
-  {% endtrans %}
   <div class="loader"></div>
-{% elif state == "verification_failed" %}
-  {% trans %}
-  Failed to send confirmation email. Please try again.
-  If you no longer have access to your email address, please update your mozilla account email address.
-  <a
-    href="https://support.mozilla.org/en-US/kb/change-primary-email-address-firefox-accounts"
-    target="_blank"
-  >
-    Change email address
-  </a>
-  {% endtrans %}
+    {% trans %}
+    Working... Please be patient.
```

Comment:
I am not sure what "working" means, is this when we are triggering a confirmation email? If so:

_We are sending an email to you, this might take a minute. The page will automatically refresh._

Or is this while the page is loading and we are fetching all the data? Then:

_We are currently fetching records for your email address, the page will refresh automatically._

Reply:
It is both. Updated with the first one.

Reply:
Updated.